curvefs/src/metaserver/s3/metaserver_s3_adaptor.h

```diff
@@ -151,6 +166,7 @@ class S3ClientAdaptorImpl : public S3ClientAdaptor {
     uint64_t batchSize_;
     uint32_t objectPrefix_;
     bool enableBatchDelete_;
+    std::unique_ptr<S3ClientAdaptorMetric> metric_;
```

Comment:
no need to save metric as a pointer, just `S3ClientAdaptorMetric metric_`?

Reply:
> no need to save metric as a pointer, just `S3ClientAdaptorMetric metric_`?

fix